packages/one-app-bundler/webpack/plugins/restrict-runtime-symbols.js

```diff
@@ -0,0 +1,34 @@
+import fs from 'node:fs';
+import {
+  validateAppConfig,
+} from '@americanexpress/one-app-dev-bundler';
+
+class RestrictRuntimeSymbols {
+  // eslint-disable-next-line class-methods-use-this -- no need for this
+  apply(compiler) {
+    compiler.hooks.assetEmitted.tap('RestrictRuntimeSymbols', (file, { targetPath }) => {
+      if (targetPath.endsWith('.node.js')) {
+        const initialContent = fs.readFileSync(targetPath, 'utf8');
+        if (initialContent.match(/create-react-class/)) {
+          throw new Error('`create-react-class` is restricted from being used');
+        }
+      } else if (targetPath.endsWith('rowser.js')) { // catch browser and legacyBrowser
```

Comment:
```js
 else if (targetPath.endsWith('rowser.js'))
```
did you mean 
```js
 else if (targetPath.endsWith('browser.js'))
```
?

Reply:
No, as the comment mentions, `browser.js` does not match `legacyBrowser.js`

Reply:
oh i completely missed the comment, makes sense